package.json

```diff
@@ -3,7 +3,7 @@
     "displayName": "Dictionary Completion",
     "description": "Word Completion",
     "icon": "images/dictionary.png",
-    "version": "1.2.2",
+    "version": "1.2.3",
```

Comment:
We usually do not update the version number in a feature commit. Just leave it to me and I will publish a new version after this PR is merged in.